src/app/tl-header/tl-header.component.scss

```diff
@@ -403,6 +403,22 @@
     left: 0;
     z-index: 1100;
     transition: width 0.5s ease-in-out, opacity 0.5s ease-in-out;
+    div {
```

Comment:
Let's add a class to the wvr component and target that.

Reply:
mobile-menu-content class added to wvr header component and that is targeted.